src/configure/configure.ts

```diff
@@ -186,7 +186,17 @@ class Orchestrator {
 
                 if (this.inputs.targetResource.resource && this.inputs.targetResource.resource.id) {
                     this.context['resourceId'] = this.inputs.targetResource.resource.id;
+                    // Getting cluster related properties for AKS scenario
+                    if (this.inputs.targetResource.resource.type === TargetResourceType.AKS) {
```

Comment:
this is specific to AKS can be make it generic to get the various properties of a resource.